README.md

```diff
@@ -75,6 +80,26 @@ if ($infusionsoft->getToken()) {
 }
 ```
 
+### API Keys
+
+API Keys are a "password" for your data in an application and should always be treated like a dangerous secret. 
+
+In our UI you will find an API Settings screen which divides API Keys into two distinct categories:
+* Personal Access Tokens, which are scoped to your own user account and can only see and manipulate the data you have access to
+* Service Account Keys, which can only be authorized by an Admin and have full access to the data stored in the application
+
+For additional information on how to authorize and use PATs and SAKs please see our [developer documentation](https://developer.infusionsoft.com/pat-and-sak/).
```

Comment:
```suggestion
For additional information on how to authorize and use PATs and SAKs please see our [developer documentation](https://developer.keap.com/pat-and-sak/).
```

Reply:
Those seem to be slightly different? WDYT? Can we use that?

Reply:
It's part of the pain and weirdness of how the developer sites are set up right now on Wordpress.  You can use either for direct links due to the nginx rewrite rules, but internal redirects will end up on the Infusionsoft version until we get a rework done.

Reply:
😢 Makes sense, sadly!!!